docs/source/deployment/kubeflow.md

```diff
@@ -15,10 +15,10 @@ Here are the main reasons to use Kubeflow Pipelines:
 
 To use Kubeflow Pipelines, make sure you have the following prerequisites in place:
 
-- Kubeflow Pipelines is [installed](https://www.kubeflow.org/docs/started/getting-started/) on your Kubernetes cluster
-- Kubeflow Pipelines SDK is [installed](https://www.kubeflow.org/docs/pipelines/sdk/install-sdk/) locally
-- A `name` attribute is set for each Kedro [node](/kedro.pipeline.node) since it is used to trigger runs
-- All node input/output DataSets must be configured in `catalog.yml` and refer to an external location (e.g. [AWS S3](../data/data_catalog.md#using-the-data-catalog-with-the-yaml-api)); you cannot use the `MemoryDataSet` in your workflow
+- [Kubeflow Pipelines is installed](https://www.kubeflow.org/docs/started/getting-started/) on your Kubernetes cluster
+- [Kubeflow Pipelines SDK is installed](https://www.kubeflow.org/docs/pipelines/sdk/install-sdk/) locally
+- A `name` attribute is set for each [Kedro node](/kedro.pipeline.node) since it is used to trigger runs
```

Comment:
A `name` attribute is set for each [Kedro node](/kedro.pipeline.node), since it is used to trigger runs